packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -913,7 +958,7 @@ export class TeamsMeetingLinkProvider implements ChatThreadProvider {
 /**
  * Arguments for use in {@link createAzureCommunicationCallWithChatAdapter} to join a Teams meeting using meeting id.
  *
- * @public
+ * @private
  */
 export class TeamsMeetingIdProvider implements ChatThreadProvider {
```

Comment:
Just a thought (can ignore this for this PR). We are going to do the logic of lazy chat for all Teams Interop join methods.
Perhaps instead of `TeamsMeetingIdProvider` and `TeamsMeetingLinkProvider` we just have a single `TeamsInteropChatProvider`?